.pylintrc

```diff
@@ -1,7 +1,7 @@
 [MASTER]
 # Add files or directories to the blacklist. They should be base names, not
 # paths.
-ignore=CVS,altgraph,macholib,pefile.py,six.py,unittest2,ordlookup,modulegraph
+ignore=CVS,altgraph,macholib,pefile.py,six.py,ordlookup,modulegraph
```

Comment:
Nice, obviously it's time to clean this up, but then we clean up all outdated stuff: pefile.py and six.py are gone, too, CVS we never had.

And please put this into a separate commit, since it is not related to dropping support for any python version. Even unittest2 is not part of PyInstaller since long now.